src/main/java/team/themoment/hellogsmv3/domain/oneseo/service/ModifyOneseoService.java

```diff
@@ -0,0 +1,131 @@
+        return OneseoPrivacyDetail.builder()
+                .id(oneseoPrivacyDetail.getId())
+                .oneseo(oneseo)
+                .graduationType(reqDto.graduationType())
+                .address(reqDto.address())
+                .detailAddress(reqDto.detailAddress())
+                .profileImg(reqDto.profileImg())
+                .guardianName(reqDto.guardianName())
+                .guardianPhoneNumber(reqDto.guardianPhoneNumber())
+                .relationshipWithGuardian(reqDto.relationWithApplicant())
+                .schoolAddress(reqDto.schoolAddress())
+                .schoolName(reqDto.schoolName())
+                .schoolTeacherName(reqDto.teacherName())
+                .schoolTeacherPhoneNumber(reqDto.teacherPhoneNumber()).build();
+    }
+
+    private MiddleSchoolAchievement buildMiddleSchoolAchievement(OneseoReqDto reqDto, MiddleSchoolAchievement middleSchoolAchievement, Oneseo oneseo) {
+        return MiddleSchoolAchievement.builder()
+                .id(middleSchoolAchievement.getId())
+                .oneseo(oneseo)
+                .transcript(reqDto.middleSchoolGrade())
+                .percentileRank(BigDecimal.ONE)
+                .totalScore(BigDecimal.ONE)
+                .artisticScore(BigDecimal.ONE)
+                .attendanceScore(BigDecimal.ONE)
+                .volunteerScore(BigDecimal.ONE)
+                .curricularSubtotalScore(BigDecimal.ONE)
+                .extraCurricularSubtotalScore(BigDecimal.ONE)
+                .gedMaxScore(BigDecimal.ONE)
+                .gedTotalScore(BigDecimal.ONE)
+                .grade1Semester1Score(BigDecimal.ONE)
+                .grade1Semester2Score(BigDecimal.ONE)
+                .grade2Semester1Score(BigDecimal.ONE)
+                .grade2Semester2Score(BigDecimal.ONE)
+                .grade3Semester1Score(BigDecimal.ONE).build();
+    }
+
+    private void saveModifiedOneseo(Oneseo modifiedOneseo, OneseoPrivacyDetail modifiedOneseoPrivacyDetail, MiddleSchoolAchievement modifiedMiddleSchoolAchievement) {
+        oneseoRepository.save(modifiedOneseo);
+        oneseoPrivacyDetailRepository.save(modifiedOneseoPrivacyDetail);
+        middleSchoolAchievementRepository.save(modifiedMiddleSchoolAchievement);
+    }
+
+    private void isNotFinalSubmitted(boolean isAdmin, Oneseo oneseo) {
+        if (!isAdmin && oneseo.getFinalSubmittedYn().equals(YES))
+            throw new ExpectedException("최종제출이 완료된 원서는 수정할 수 없습니다.", HttpStatus.BAD_REQUEST);
+    }
+
+    private void ifScreeningChangeSaveHistory(Screening beforeScreening, Oneseo oneseo) {
+        if (oneseo.getAppliedScreening() != beforeScreening) {
+            ScreeningChangeHistory screeningChangeHistory = ScreeningChangeHistory.builder()
+                    .beforeScreening(oneseo.getAppliedScreening())
+                    .afterScreening(beforeScreening)
+                    .oneseo(oneseo).build();
+
```

Comment:
```suggestion
                    .afterScreening(beforeScreening)
                    .oneseo(oneseo)
                    .build();

```
기존 코드와 포맷이 일치하도록 변경하는 게 좋아 보여요.